metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -35,7 +35,20 @@
 DEFAULT_ENV = DEFAULT_ENV_CONFIGURATION
 DEFAULT_FLOW_CLUSTER = "prod"
 UNKNOWN_USER = "urn:li:corpuser:unknown"
-
+SQL_STYLE_PLATFORMS: Set[str] = {
+    "athena",
+    "bigquery",
+    "druid",
+    "hive",
+    "mariadb",
+    "mssql",
+    "mysql",
+    "oracle",
+    "postgres",
+    "redshift",
+    "snowflake",
+    "trino",
+}
```

Comment:
is there any more generic way to specify sql systems? What's the plan for making sure this set stays updated?

Reply:
So long as we use the mce_builder to mint urns, there is no fool-proof way to ensure this, except via code-reviews. The other way would be to provide an urn minting factory that is customizable per source. That's a much bigger change, and we should do it at some point in future.